src/components/screens/settings-sound-screen.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react"
+import ReactPlayer from 'react-player'
+
+import BasicScreen from './basic-screen';
+
+import {useHistory} from "react-router-dom";
+import {useTranslation} from 'react-i18next';
+
+import type { Node } from 'react'
+
+import sound from "../../images/settings/sound.png";
+import testSound from "../../sounds/testSound.ogg";
+import './settings-sound-screen.css'
+import {Button, Image} from "semantic-ui-react";
```

Comment:
Importy powinny być w  miarę poszeregowane, musimy też pamiętać o spójnym używaniu cudzysłowiów.
  
```suggestion
import React, { useState } from "react";
import {Button, Image} from "semantic-ui-react";
import ReactPlayer from "react-player";

import {useHistory} from "react-router-dom";
import {useTranslation} from "react-i18next";

import BasicScreen from "./basic-screen";

import sound from "../../images/settings/sound.png";
import testSound from "../../sounds/testSound.ogg";

import "./settings-sound-screen.css";
import type { Node } from "react";
```